tasks/validate-example.js

```diff
@@ -40,7 +40,7 @@ module.exports = function() {
       // skip over experiments which will fail validation
       if (file.metadata &&
         (file.metadata.experiments || file.metadata.skipValidation) ||
-         !file.path.endsWith('.html')) {
+         !file.path.endsWith('.html') || file.path.includes('live_blog')) {
```

Comment:
please don't hardcode samples - use metadata instead. If you look at the code you will see that there is already a flag for skipping sample validation...